k2/docs/source/conf.py

```diff
@@ -12,6 +12,8 @@
 #
```

Comment:
How about we move the docs to (top-level)/docs/, and remove the old docs?

Reply:
Sure, will move it in the next pullrequest.